resources/js/features/redirect/components/+root/RedirectRoot.test.tsx

```diff
@@ -0,0 +1,42 @@
+import { expect } from 'vitest';
```

Comment:
nit, consistency: We have these vitest imports configured as globals. This line is safe to remove.